docs/ad/index.rst

```diff
@@ -0,0 +1,42 @@
+.. include:: ad_attention.inc
+
+.. _ad:
+
+==============================
+Analysis & Diagnostic Toolkits
+==============================
+
+Analyses and diagnostics go hand-in-hand, but have subtle differences.  Thus,
+PlasmaPy gives each their own sub-packages, `plasmapy.analysis` and
+`plasmapy.diagnostics` respectively.
+
+
+Think of the `plasmapy.analysis` as your toolbox.  It has all the tools
+(functionality) you need to analyze your data.  Functionality is built around
+`numpy` arrays and each function has a well-defined, focused task.
```

Comment:
A possibility: link to `numpy.ndarray` instead of NumPy

Reply:
I made this a link to `numpy`'s page about their [array objects](https://numpy.org/doc/stable/reference/arrays.html).